src/platform_impl/macos/menu.rs

```diff
@@ -497,18 +511,37 @@ impl Accelerator {
       KeyCode::KeyX => "x".into(),
       KeyCode::KeyY => "y".into(),
       KeyCode::KeyZ => "z".into(),
-      KeyCode::Digit0 => "0".into(),
-      KeyCode::Digit1 => "1".into(),
-      KeyCode::Digit2 => "2".into(),
-      KeyCode::Digit3 => "3".into(),
-      KeyCode::Digit4 => "4".into(),
-      KeyCode::Digit5 => "5".into(),
-      KeyCode::Digit6 => "6".into(),
-      KeyCode::Digit7 => "7".into(),
-      KeyCode::Digit8 => "8".into(),
-      KeyCode::Digit9 => "9".into(),
-      KeyCode::Escape => "\u{1b}".into(),
+      KeyCode::Minus => "-".into(),
+      KeyCode::Period => ".".into(),
+      KeyCode::Quote => "'".into(),
+      KeyCode::Semicolon => ";".into(),
+      KeyCode::Slash => "/".into(),
       KeyCode::Space => "\u{0020}".into(),
+      KeyCode::Tab => "⇥".into(),
+      KeyCode::Escape => "\u{001b}".into(),
+      KeyCode::F13 => "\u{F710}".into(),
+      KeyCode::F14 => "\u{F711}".into(),
+      KeyCode::F15 => "\u{F712}".into(),
+      KeyCode::F16 => "\u{F713}".into(),
+      KeyCode::F17 => "\u{F714}".into(),
+      KeyCode::F18 => "\u{F715}".into(),
+      KeyCode::F19 => "\u{F716}".into(),
+      KeyCode::F20 => "\u{F717}".into(),
+      KeyCode::F21 => "\u{F718}".into(),
+      KeyCode::F22 => "\u{F719}".into(),
+      KeyCode::F23 => "\u{F71A}".into(),
+      KeyCode::F24 => "\u{F71B}".into(),
+      KeyCode::F25 => "\u{F71C}".into(),
+      KeyCode::F26 => "\u{F71D}".into(),
+      KeyCode::F27 => "\u{F71E}".into(),
+      KeyCode::F28 => "\u{F71F}".into(),
+      KeyCode::F29 => "\u{F720}".into(),
+      KeyCode::F30 => "\u{F721}".into(),
+      KeyCode::F31 => "\u{F722}".into(),
+      KeyCode::F32 => "\u{F723}".into(),
+      KeyCode::F33 => "\u{F724}".into(),
+      KeyCode::F34 => "\u{F725}".into(),
+      KeyCode::F35 => "\u{F726}".into(),
```

Comment:
these should be with other F keys below

Reply:
Should I remove the `from NSText.h` and `from NSEvent.h` comments in that case?

Reply:
no, lets keep them, I am sure david added them for a reason.